images/base/files/etc/crictl.yaml

```diff
@@ -1 +1 @@
-runtime-endpoint: unix:///var/run/containerd/containerd.sock
\ No newline at end of file
+runtime-endpoint: unix:///run/containerd/containerd.sock
```

Comment:
heh, didn't know that
```
root@kind-control-plane:/# ls -al /var/run
lrwxrwxrwx 1 root root 4 Apr 10 20:34 /var/run -> /run
```

Reply:
Yeah, in our case and on most current distros I would expect them to be the same BUT I checked and containerd appears to nominally use /run canonically.

Lack of unix:// has left warnings in the kubelet logs.